rcldotnet/rcldotnet.c

```diff
@@ -37,14 +39,33 @@ const char *native_rcl_get_rmw_identifier() {
   return rmw_get_implementation_identifier();
 }
 
+const char * native_rcl_get_error_string() {
+  return rcl_get_error_string();
+}
+
+const char * native_rmw_get_error_string() {
+  return rmw_get_error_string();
+}
+
 const char *native_rcl_get_error_string_safe() {
   return rcl_get_error_string_safe();
 }
 
 void native_rcl_reset_error() { rcl_reset_error(); }
+void native_rmw_reset_error() { rmw_reset_error(); }
 
 bool native_rcl_ok() { return rcl_ok(); }
 
+
+int32_t native_rmw_validate_node_name(const char *name) {
```

Comment:
This goes against the expected semantics of `rmw_validate_node_name`. The original API takes `validation_result` as a pointer to an integer and returns a `rmw_ret`. It'd be slightly better if this followed the same API, but it's ok if not. Please add a comment explaining that this function has different semantics.

Reply:
Fixed. Now the API is the same of the original